### PR TITLE
Convert tf.stack, tf.tile, tf.min, etc via keras _builtin

### PR DESCRIPTION
**Pull request by jiafatom**

### CODE REVIEW

File: keras2onnx/_builtin.py [hunk 1]
```diff
@@ -317,6 +394,18 @@ def convert_tf_squeeze(scope, operator, container):
                               axes=axis)
 
 
+@converter_func(TYPES.Tile)
+def convert_tf_tile(scope, operator, container):
+    oopb = OnnxOperatorBuilder(container, scope)
+    cast_1 = oopb.add_node('Cast',
```
Comment (vinitra) [inline keras2onnx/_builtin.py]: Why do we need a cast op here for tile?

File: tests/test_layers.py [hunk 2]
```diff
@@ -193,6 +206,29 @@ def test_tf_squeeze(self):
             expected = model.predict(data)
             self.assertTrue(run_onnx_runtime('onnx_squeeze', onnx_model, data, expected, self.model_files))
 
+    def test_tf_stack(self):
+        def myFunc1(x):
```
Comment (vinitra) [inline tests/test_layers.py]: nit: consistent naming scheme with underscores, call this function test_func_1 or my_func_1

File: tests/test_layers.py [hunk 3]
```diff
@@ -193,6 +206,29 @@ def test_tf_squeeze(self):
...
+            return tf.stack([x[0], x[1], x[2]], axis=1)
+
+        def myFunc2(x):
```
Comment (vinitra) [inline tests/test_layers.py]: likewise, my_func_2

File: keras2onnx/_builtin.py (hunk 1 above)
Comment (jiafatom) [reply keras2onnx/_builtin.py]: onnx Tile input 1 needs to be int64.

**Pull request merged**